feature-detects/flash.js

```diff
@@ -77,11 +85,18 @@ define(['Modernizr', 'createElement', 'docElement', 'addTest', 'getBody'], funct
           }
         }
 
-        // If we’re rockin’ a fake body, clean it up
-        if (body.fake) {
+        // If we’re rockin’ an attached fake body, clean it up
+        if (body.fake && body.parentNode) {
```

Comment:
Seeing the same thing, so I created [pull request](https://github.com/Modernizr/Modernizr/pull/1506) for this.